frontend/src/app/main/ui/dashboard/files.cljs

```diff
@@ -158,7 +158,7 @@
          (mf/deps project)
          (fn [origin]
            (let [mdata  {:on-success on-file-created}
-                 params {:project-id (:id project)}]
+                 params {:project-id (:id project) :has-files (> file-count 0)}]
```

Comment:
This should not be a parameter, because it is not necessary as parameter. This should go directly on metadata, in the same way as ::ev/origin is passed some lines below